platforms/qurt/src/px4/SerialImpl.cpp

```diff
@@ -276,6 +276,23 @@ ssize_t SerialImpl::write(const void *buffer, size_t buffer_size)
 	return ret_write;
 }
 
+ssize_t SerialImpl::writeBlocking(const void *buffer, size_t buffer_size, uint32_t timeout_ms)
```

Comment:
This looks fine. qurt_uart_write is blocking by definition so will need to look at implementing a non-blocking write. I'll address this in a separate PR.